resources/views/livewire/stream-list.blade.php

```diff
@@ -1,5 +1,22 @@
 <div class="py-16 space-y-16" id="scrollTop">
-    @foreach ($streamsByDate as $date => $streams)
+    @if($isArchive)
+        <div class="w-full max-w-xl px-4 mx-auto sm:px-6 md:px-8">
+            <div>
+                <label for="search" class="sr-only">Search</label>
+                <div class="mt-1 relative rounded-md shadow-sm">
+                    <div class="absolute inset-y-0 left-0 pl-3 flex items-center pointer-events-none" aria-hidden="true">
+                        <!-- Heroicon name: solid/search -->
+                        <svg class="mr-3 h-4 w-4 text-gray-400" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
+                            <path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd" />
+                        </svg>
+                    </div>
+                    <input wire:model.debounce.300ms="search" type="search" name="search" id="search" class="text-gray-600 focus:ring-indigo-500 focus:border-indigo-500 block w-full pl-9 sm:text-sm border-gray-300 rounded-md" placeholder="Search">
```

Comment:
Please use a red focus ring, similar to here https://github.com/christophrumpel/larastreamers/blob/main/resources/views/livewire/submit-you-tube-live-stream.blade.php#L30